index.js

```diff
@@ -4,7 +4,7 @@ import Fab from './Fab';
 import Icon from './Icon';
 import IconButton from './IconButton';
 import IconToggle from './IconToggle';
-import Switch from './Switch';
+import Switch from './ts/Switch/index';
```

Comment:
this should not change, else it'll not be backward compatible

Reply:
Sorry, my IDE does this automatically, when I'm moving files will fix those issues at the end